Makefile

```diff
@@ -51,3 +51,10 @@ generate-game-client-protos:
 		--js_out=import_style=commonjs:apps/game_client/assets/js/protobuf \
 		--proto_path=apps/serialization \
 		messages.proto
+
+generate-arena-load-test-protos:
```

Comment:
Shouldn't we add this new command to `generate-protos`?

Reply:
Yeah. done!